introduction.md

```diff
@@ -0,0 +1 @@
+Hi ! My name is Pascal, I come from France and I'm 26 years old.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.